SayTheirNames/Source/Controller/Petitions/PetitionTableViewCell.swift

```diff
@@ -0,0 +1,218 @@
+        label.backgroundColor = Self.VerifiedLabelBackgroundColor
+        label.textColor = Self.VerifiedLabelTextColor
+        return label
+    }()
+    
+    private lazy var titleLabel: UILabel = {
+        let label = UILabel()
+        label.font = UIFont.STN.bannerTitle
+        label.textColor = Self.ContentColor
+        label.numberOfLines = 1
+        label.lineBreakMode = .byTruncatingTail
+        return label
+    }()
+
+    private lazy var summaryLabel: UILabel = {
+        let label = UILabel()
+        label.font = UIFont.STN.summary
+        label.textColor = Self.ContentColor
+        label.numberOfLines = 3
+        label.lineBreakMode = .byTruncatingTail
+        return label
+    }()
+
+    private lazy var findOutMoreButton: UIButton = {
+        let button = UIButton(type: .custom)
+        button.addTarget(self, action: #selector(findOutMoreButtonTapped), for: .touchUpInside)
+        
+        button.setTitle(Self.FindOutMoreButtonTitle.uppercased(), for: .normal)
+        button.layer.borderColor = Self.ContentColor.cgColor
+        button.titleLabel?.font = UIFont.STN.sectionHeader
+        button.layer.borderWidth = Self.FindOutMoreButtonOutlineThickness
+        button.tintColor = Self.ContentColor
+        button.setTitleColor(Self.ContentColor, for: .normal)
+        return button
+    }()
+        
+    private var hasLayedOutSubviews = false
+    private var allConstriants: [NSLayoutConstraint] = []
+    private func createLayout() {
+        guard !hasLayedOutSubviews else { return }
+        hasLayedOutSubviews = true
+
+        [container,
+         imageViewContainer,
+         bannerImageView,
+         verifiedLabel,
+         titleLabel,
+         summaryLabel,
+         findOutMoreButton].forEach {
+            $0.translatesAutoresizingMaskIntoConstraints = false
+        }
+        
+        contentView.addSubview(container)
+        
+        imageViewContainer.addSubview(verifiedLabel)
+        
+        [imageViewContainer,
+         titleLabel,
+         summaryLabel,
+         findOutMoreButton,
+         verifiedLabel].forEach {
+            container.addSubview($0)
+        }
+        
+        // the verified label should always appear visually above everything else
+        verifiedLabel.layer.zPosition = 100
+    }
+    
+    // ensure that updateConstraints is always called
+    override class var requiresConstraintBasedLayout: Bool { true }
+    
+    override func updateConstraints() {
+        super.updateConstraints()
+                
+        createLayout()
+        
+        // NOTE: when changing from landscape to portrait, or vice versa,
+        // this code DOES cause autolayout conflicts
+        // but they are fixed in subsequent passes
+        // in the same call stack and thus are never seen by the user
+        
+        titleLabel.setContentCompressionResistancePriority(.defaultLow - 1, for: .horizontal)
+        
+        NSLayoutConstraint.deactivate(allConstriants)
```

Comment:
it's explicitly mentioned in the documentation not to deactivate all constraints and reactivate them as it greatly affects performance: https://developer.apple.com/documentation/uikit/uiview/1622512-updateconstraints

it's a pretty common practice to store a bool to only do it once, or to do it on init.
there usually shouldn't be any issues, so if constraints break then there's something else going on. 

if you're unable to spend time on fixing it in this PR, please create a trello card 🙏

Reply:
fixed, thanks